### DISCUSSION TITLE
各位，我想润，有些问题想请教下。

### DISCUSSION THREAD

**Discussion by Casuwin** [The-Run-Philosophy-Organization/run#182] [General]
我GPA大概只有３，工作了有一年多了。感觉润出去不知道能找到什么工作，学习是不可能学习的，不太可能再进学校了，没那门路也没钱。所以我想问问假如润美国的话，我去当个洗车工能挣多少啊，或者有哪些好的机械厂推荐。

另外，我喜欢做菜吃东西，想问问美国那边物价怎么样。
－　３０斤大米要多少钱？没大米的话３０斤面粉呢？
－　买条３斤重的草鱼或者剥皮鱼要多少钱？
－　还喜欢抽烟，那边有什么烟，一条多少钱啊？
－　刚去那里没医疗保险，一盒感冒药，胃药，还有痔疮膏要多少钱？
－　一箱黑啤，或者麦啤要多少钱？
－　另外听说那边水电费都是不同的公司的，有哪些便宜的水电费区域？
－　听我同学说一套洗剪吹要３０美元，有什么便宜的连锁理发店吗？
－　苹果手机在美国便宜，那我可以不维修直接换新的吗？
－　同学上次去美国，说坐公交难，又要等还会抛锚，那我得买辆车，只要求代个步还能搬东西，买辆小面包吧，要多少钱啊？或者买辆大皮卡更好？我看电影里都是大皮卡．

I am looking forward to hearing from you.

**Comment by Cheng-liuliu** [comment]
你好，请问你现在国内做什么工作？有兴趣一起去美国洗车吗？带我👀

**Reply by Casuwin** [reply]
在国内做牛马呢．
所以想润美国去，一起啊，不过先等等作者回答问题吧．
毕竟我没啥资本，没试错机会啊．

**Comment by Hzsen** [comment]
大皮卡比一般的二手跑车还贵...理发的话我带了个质量好点的推子然后留长发就基本没去过理发店，估计太长了才会去（当然能不能留长发比较看脸型）。物价最近通胀有点明显，几乎有50%了属于是。电子产品普遍便宜，经常打折。

**Reply by Casuwin** [reply]
嗯,还有吗?米菜药有说道吗?

**Comment by stackunderflow111** [comment]
你想想你想怎么润美国，偷渡吗？
黑下来的华人一般都是干老三样，餐馆/装修/按摩。洗车需要证书我记得，所以你别想了。

**Reply by Cheng-liuliu** [reply]
2022年，比起上个10年，还好黑下来吗？

**Reply by stackunderflow111** [reply]
美国的政策没有明显变化呀

**Comment by stackunderflow111** [comment]
其他问题我挨个回答一下。

－　３０斤大米要多少钱？没大米的话３０斤面粉呢？（这种是我比较常买的米 https://www.amazon.com/Nishiki-Premium-Medium-Grain-15-Pound/dp/B004NRLAVY/?th=1 一包15磅20刀）
－　买条３斤重的草鱼或者剥皮鱼要多少钱？（不知道哪里有淡水鱼买，超市里面我只看得到海鱼）
－　还喜欢抽烟，那边有什么烟，一条多少钱啊？（不抽烟）
－　刚去那里没医疗保险，一盒感冒药，胃药，还有痔疮膏要多少钱？（https://www.goodrx.com/ 这个网站输入你想买的药的名字，可以给你优惠券，拿着优惠券去周围药店买会打很多折。你可以自己查一查你需要买的药）
－　一箱黑啤，或者麦啤要多少钱？（没买过）
－　另外听说那边水电费都是不同的公司的，有哪些便宜的水电费区域？（租房，公寓包了）
－　听我同学说一套洗剪吹要３０美元，有什么便宜的连锁理发店吗？（连锁的都不便宜，便宜的你可以去越南人开的理发店，大部分都是打黑工的越南人，一次十美元左右）
－　苹果手机在美国便宜，那我可以不维修直接换新的吗？（我建议你多花钱在生活必需品上）
－　同学上次去美国，说坐公交难，又要等还会抛锚，那我得买辆车，只要求代个步还能搬东西，买辆小面包吧，要多少钱啊？或者买辆大皮卡更好？我看电影里都是大皮卡．（大城市公交其实还可以。买车的话可以买N手的，不太贵。价格可以网上看，google buy a used car）
